cuda/test/components/prefix_sum.cu

```diff
@@ -44,6 +41,9 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <ginkgo/core/base/array.hpp>
 
 
+#include "core/components/prefix_sum.hpp"
```

Comment:
I would prefer to leave this header at the top since it is the component under test